server/consumer.go

```diff
@@ -33,15 +33,16 @@ import (
 )
 
 type ConsumerInfo struct {
-	Stream         string         `json:"stream_name"`
-	Name           string         `json:"name"`
-	Created        time.Time      `json:"created"`
-	Config         ConsumerConfig `json:"config"`
-	Delivered      SequencePair   `json:"delivered"`
-	AckFloor       SequencePair   `json:"ack_floor"`
-	NumPending     int            `json:"num_pending"`
-	NumRedelivered int            `json:"num_redelivered"`
-	NumWaiting     int            `json:"num_waiting"`
+	Stream           string         `json:"stream_name"`
+	Name             string         `json:"name"`
+	Created          time.Time      `json:"created"`
+	Config           ConsumerConfig `json:"config"`
+	Delivered        SequencePair   `json:"delivered"`
+	AckFloor         SequencePair   `json:"ack_floor"`
+	NumPending       int            `json:"num_pending"`
+	NumRedelivered   int            `json:"num_redelivered"`
+	NumWaiting       int            `json:"num_waiting"`
+	NumStreamPending uint64         `json:"num_stream_pending"`
```

Comment:
hmm, not sure I think the name of this is confusing between num_pending and num_stream_pending. Maybe its num_pending thats confusing but together its very confusing.

Reply:
Agree but we had that one already, NumPending. That is really NumAckPending and the NumStreamPending is NumPendning for me.